### PR TITLE
Use case + user story + UML diagram

### PR DESCRIPTION
**Pull request by ChryslineLim**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -669,23 +675,23 @@ automation of matching between suitable properties and buyers
...
+| `* * *`  | property agent                                 | add new buyers                                                                        |                                                                   |
+| `* * *`  | property agent                                 | add new properties                                                                    |                                                                   |
+| `* *`    | property agent                                 | edit contact and property information offline                                         | keep the information updated                                      |
+| `* *`    | property agent                                 | delete existing contact and propety entries                                           | remove redundant information, to keep dataset neat                |
+| `* *`    | property agent with a large number of contacts | sort buyers and properties according to various relevant criteria                     | easily manipulate my list in a way I want to view the information |
```
Comment (riccqi) [inline docs/DeveloperGuide.md:684]:
```suggestion
| `* *`    | property agent with a large number of contacts | sort buyers and properties according to various relevant criteria                     | easily find information using their order |
```

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -784,6 +790,33 @@ Use case ends.
...
+2. User executes sort buyers command on the currently displayed buyer list.
+
+Use case ends.
+
+### Use case: List buyers
+
+**MSS:**
+
+1. User chooses to list buyers
+2. User executes list buyers command.
```
Comment (riccqi) [inline docs/DeveloperGuide.md:816]: 3. Cobb shows a list of all the buyers currently saved?

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -784,6 +790,33 @@ Use case ends.
...
+2. User executes sort properties command on the currently displayed properties list.
+
+Use case ends.
+
+### Use case: Sort buyers
+
+**MSS:**
+
+1. User chooses to sort buyers by a specified field in a specified order.
+2. User executes sort buyers command on the currently displayed buyer list.
```
Comment (riccqi) [inline docs/DeveloperGuide.md:807]: There should be a step 3 indicating the success outcome

File: docs/diagrams/SortSequenceDiagram.puml [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+ArrayList -> Model
+deactivate ArrayList
+
+Model -> BuyerBook : setBuyers(sortedList)
+activate BuyerBook
+
+BuyerBook -> UniqueBuyerList : setBuyers(buyerList)
+activate UniqueBuyerList
+
+UniqueBuyerList -> ObservableArrayList : setAll(internalList)
```
Comment (riccqi) [inline docs/diagrams/SortSequenceDiagram.puml:43]: Isn't the thing just called `ObservableList`

**Review by riccqi — APPROVED** [review]
Some changes needed for the use cases section. LGTM otherwise

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (ChryslineLim) [reply docs/DeveloperGuide.md:816]: Oops I accidentally copied this from on top, but i'll help to change that!

File: docs/diagrams/SortSequenceDiagram.puml (hunk 4 above)
Comment (ChryslineLim) [reply docs/diagrams/SortSequenceDiagram.puml:43]: You're right.... oops

**Pull request merged**